### ISSUE TITLE
Unable to delete character/remove character from world after transfer

### ISSUE THREAD

**Issue by awildenhain** [worldanvil/worldanvil-bug-tracker#1166]
**World Anvil Username**: 
Nonopherian 
Associated with Ticket #33 in your ticketing system (I talked with Madeline about this issue on July 10, 2022)

**World / Campaign / RPG Character / Manuscript Title**: 
Soul of the Sea Campaign (private)

**Feature**: 
Characters/Heroes, character transfer

**Related URLS** (To public & visible pages add screenshots otherwise): 
Private world and campaign, see screenshots below.
Character that needs to be deleted is here: https://www.worldanvil.com/hero/7968181a-dc27-4543-becc-2061e81d5515

**Describe the Issue**
TLDR: My brother and I (two separate World Anvil accounts) seem to have glitched a character that we transferred from my account to his, and were hoping you could:
1. Delete it for us (immediate short-term solution)
2. Address the bug that caused it to be this way (long-term solution)

**To Reproduce**
1. Make sure you are using a World Anvil account that has an associated D&D 5e Campaign
2. Create a character and a linked D&D 5e (2020) Character sheet
3. Transfer that character over to another World Anvil account
4. From the second World Anvil account, attempt to join the Campaign owned by the first account.
**Result 1:**  The second account receives a 404 error and is unable to join the Campaign.  The character is not part of the campaign or part of the party.  However, if the owner of the campaign goes to the Parties tab of the campaign, the portrait of the character appears in the party (screenshot 1).  If the owner of the campaign clicks Edit Party and navigates to Settings, the character does not appear in the Characters tag box.  The character seems to be part of the party in some areas and not in others, and there is no way to remove them from the party fully.
5. From the second account, navigate to the Character (the name of the character with the eyeball icon next to it, to see the main character page).
6. Click the edit button
7. Expand Delete Character
**Result 2:**  The delete button is absent from the Delete Character accordion (screenshot 3).  User is unable to delete the character.
8. Expand Transfer Character
9. Type in the username of the first user (the one who created the character and owns the campaign), and click Transfer Character
10. Log in as the first user and navigate to the Character (the name of the character with the eyeball icon next to it, to see the main character page).
11. Click the edit button
12. Expand Delete Character
**Result 3:**  The delete button is absent from the Delete Character accordion (screenshot 3).  User is unable to delete the character.

**Expected behavior**
1. Someone should be able to delete the character at all times, regardless of where it originated
3. Characters should either be part of a party, or not part of a party.  They should not appear as part of a party in some areas of World Anvil and not part of a party in others.

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!
![image](https://user-images.githubusercontent.com/55631431/179418228-0e0c4891-4221-4500-a5da-cd5ef5352f18.png)
![image](https://user-images.githubusercontent.com/55631431/179418378-ddd42391-2a27-4cfe-88b8-f59f3dfd6c47.png)
![image](https://user-images.githubusercontent.com/55631431/179418551-710c893d-a6dc-4186-9b26-2612a197f41a.png)

**Comment by Kordishal** [comment]
Hi! You must have linked the wrong issue, because that one has nothing to do with this, but there is an issue like that yeah.

Now my first question is have you tried to expel and the reinvite the character to the campaign? 

If the character is not shown under Protagonists (PCs) go to the world and check in the sidebar under Advanced Tools and expel the character from the world there.

Not being able to delete a character which is part of a world is very much working as intended. There is a bug where sometimes campaign invite does not work correctly.

**Comment by awildenhain** [comment]
Hi there!  Sorry, I didn't mean to link the github issue, that's the issue number I got via email after I chatted with support, idk if it is even valuable info for you.

I am trying to find where to expel the character from the world.  You do not mean navigating to Access Rights Management/Subscribers and removing all groups from the user who plays the character, right?  I assume there is a way to expel a character and not the subscriber?
